proton

```diff
@@ -239,12 +239,16 @@ else:
 lfile = None
 if "SteamGameId" in env:
     if env["WINEDEBUG"] != "-all":
-        lfile_path = os.environ["HOME"] + "/steam-" + os.environ["SteamGameId"] + ".log"
+        lfile_path = os.environ["HOME"] + "/steam-proton-" + os.environ["SteamGameId"] + ".log"
         if os.path.exists(lfile_path):
             os.remove(lfile_path)
         lfile = open(lfile_path, "w+")
 else:
-    env["WINEDEBUG"] = "-all"
+    if env["WINEDEBUG"] != "-all":
+        lfile_path = os.environ["HOME"] + "/steam-proton-unknown" + ".log"
+        if os.path.exists(lfile_path):
```

Comment:
I think there's no need to duplicate lines 243-245 (to 249-251). You could move `lfile_path` variable to separate scope and avoid some code duplication this way:

    lfile_path = os.environ["HOME"] + "/steam-proton-unknown.log"
    appid = env.get("SteamGameId")
    if appid:
        lfile_path = os.environ["HOME"] + "/steam-proton-" + appid + ".log"
    if env["WINEDEBUG"] != "-all":
        # rest of logic follows

Also, it might be good idea to introduce `os.path.join` to proton file finally ;)

Reply:
There's probably a good bit of cleanup to do : )  Might be better to wrap that up in a separate pull for the sake of scope?
I'll try to take a look at that later on today.

Reply:
I think it's appropriate to keep it in this PR, but it's ok to put it in new commit (or you might want to amend or rebase --interactive).